_posts/commands/2020-05-01-mom_saveloc_current.md

```diff
@@ -7,4 +7,4 @@ tags:
 safeguard: mom_run_safeguard_saveloc_tele
 ---
 
-Teleports the player to their current saved location.
\ No newline at end of file
+Teleports the player to their current saved location. Binding +mom_saveloc_current and holding the set key down, freezes the player and loads the saveloc. Upon releasing the set key (-mom_saveloc_current) will unfreeze the player.
```

Comment:
Verryy minor nitpick, last one i promise!
```suggestion
Teleports the player to their current saved location. Binding +mom_saveloc_current and holding the set key down, freezes the player and loads the saveloc. Releasing the set key (-mom_saveloc_current) will unfreeze the player.
```

Reply:
The "And" in red, you want them removed?

Reply:
I think it's coloring that like it's some kind of programming logic haha. DW about it